nodeup/pkg/model/protokube.go

```diff
@@ -181,6 +181,7 @@ type ProtokubeFlags struct {
 	DNSInternalSuffix *string  `json:"dnsInternalSuffix,omitempty" flag:"dns-internal-suffix"`
 	DNSProvider       *string  `json:"dnsProvider,omitempty" flag:"dns"`
 	DNSServer         *string  `json:"dns-server,omitempty" flag:"dns-server"`
+	Image             *string  `json:"image,omitempty" flag:"image"`
```

Comment:
I think this is the etcd image, so how about `EtcdImage *string `json:"image,omitempty" flag:"etcd-image"`.  It's the flag that is hard to change and matters most.